DataFormats/Detectors/Common/src/DetMatrixCache.cxx

```diff
@@ -28,3 +28,16 @@ void DetMatrixCache::setSize(int s)
   }
   mSize = s;
 }
+
+void DetMatrixCacheIndirect::setSize(int size, int sizeIndirect)
+{
+  // set the size of the matrix cache, can be done only once
+  if (mSize != 0 || mIndirectSize != 0) {
+    LOG(FATAL) << "Cache size (N sensors) was already set to " << mSize << " / " << mIndirectSize << FairLogger::endl;
```

Comment:
note that `FairLogger::endl` is no longer needed (there is an implicit end-of-line) in `LOG`.

Reply:
Ah, I was not aware. Actually, I just copied and moved the code. I'll remove the endl everywhere.